framework/contrib/hit/include/hit/parse.h

```diff
@@ -212,6 +212,7 @@ class Node
   virtual std::vector<double> vecFloatVal();
   virtual std::vector<bool> vecBoolVal();
   virtual std::vector<int> vecIntVal();
+  virtual std::vector<int64_t> vecInt64Val();
```

Comment:
How about
```suggestion
  virtual std::vector<dof_id_type> vecDofIdTypeVal();
```

Reply:
I prefer vecInt64Val here, no need to make it libMesh-aware.

Though we should cast_int before trying to put the result in a `dof_id_type`

Reply:
I think supporting the underling raw type is preferred. This is in the contrib folder thus I suppose should be independent on libMesh defined types.